lib/exposes.js

```diff
@@ -361,7 +361,7 @@ class Climate extends Base {
 
     withSystemMode(modes, access=a.ALL) {
         assert(!this.endpoint, 'Cannot add feature after adding endpoint');
-        const allowed = ['off', 'heat', 'cool', 'auto', 'dry', 'fan_only', 'sleep'];
+        const allowed = ['off', 'heat', 'cool', 'auto', 'dry', 'fan_only', 'sleep', 'manual', 'temphand', 'holiday'];
```

Comment:
These options are limited because of HA (only allow these: https://www.home-assistant.io/integrations/climate.mqtt/#modes) Please add this to the presets instead.